app/editor/src/components/form/select/styled/SelectField.tsx

```diff
@@ -1,18 +1,30 @@
+import Select from 'react-select';
 import styled from 'styled-components';
 
-import { DropdownVariant, IDropdownProps } from '..';
+import { ISelectProps, SelectVariant } from '..';
 
-export const Dropdown = styled.div<IDropdownProps>`
-  input {
-    margin: 1px 2px 1px 2px;
-    text-decoration: ${(props) => (props.variant === DropdownVariant.link ? 'underline' : 'none')};
-    display: inline-block;
+export const SelectField = styled(Select)<ISelectProps<any>>`
```

Comment:
Use styled components for styling, don't use inline styling or custom inline styling.